analysis/tbcshaman/src/CONFIG.tsx

```diff
@@ -35,7 +35,7 @@ const config: Config = {
   builds: {
     [Build.DEFAULT]: {
       url: 'standard',
-      name: '20/41/0',
+      name: '0/5/56',
```

Comment:
that is a pretty large change in the default talent build. was there a meta shift or was this a typo, etc?

Reply:
I can only assume a copy-pasta. There isn't really a known/common 20/41/0 build for shaman at all.
![image](https://user-images.githubusercontent.com/1750811/140624832-b71d2323-4386-4e59-9c2c-23573e8e3e0e.png)
Priest however does have a 20/41/0 build.

Reply:
:+1: